src/snowcli/cli/object/show/commands.py

```diff
@@ -4,23 +4,23 @@
 
 from snowcli.cli.common.decorators import global_options_with_connection
 from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
-from snowcli.cli.warehouse.manager import WarehouseManager
+from snowcli.cli.object.show.manager import ShowManager
 from snowcli.output.decorators import with_output
 from snowcli.output.types import QueryResult
 
 app = typer.Typer(
-    name="warehouse",
+    name="show",
     context_settings=DEFAULT_CONTEXT_SETTINGS,
     help="Manages warehouses.",
 )
 
 
-@app.command("status")
+@app.command("warehouses")
 @with_output
 @global_options_with_connection
 def warehouse_status(**options):
     """
     Shows the status of each warehouse in the configured environment.
     """
-    cursor = WarehouseManager().show()
+    cursor = ShowManager().show()
```

Comment:
I think we should have more generic manager that will have the following methods:
```
def describe(object_type: str, name:str)

def drop(object_type: str, name:str)

def show(object_type: str)
```

Such commands probably can be incorporated in `SqlExecutionMixin`. WDYT?